src/pages/home/report/OnboardingReportFooterMessage.tsx

```diff
@@ -0,0 +1,98 @@
+import React, {useMemo} from 'react';
+import {View} from 'react-native';
+import {withOnyx} from 'react-native-onyx';
+import type {OnyxCollection, OnyxEntry} from 'react-native-onyx';
+import Text from '@components/Text';
+import TextLink from '@components/TextLink';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import useWindowDimensions from '@hooks/useWindowDimensions';
+import * as PolicyUtils from '@libs/PolicyUtils';
+import Navigation from '@navigation/Navigation';
+import * as ReportInstance from '@userActions/Report';
+import type {OnboardingPurposeType} from '@src/CONST';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import ROUTES from '@src/ROUTES';
+import type {Policy as PolicyType, Report} from '@src/types/onyx';
+
+type OnboardingReportFooterMessageOnyxProps = {choice: OnyxEntry<OnboardingPurposeType>; reports: OnyxCollection<Report>; policies: OnyxCollection<PolicyType>};
+type OnboardingReportFooterMessageProps = OnboardingReportFooterMessageOnyxProps;
```

Comment:
```suggestion
type OnboardingReportFooterMessageOnyxProps = {
    choice: OnyxEntry<OnboardingPurposeType>;
    reports: OnyxCollection<Report>;
    policies: OnyxCollection<PolicyType>;
};

type OnboardingReportFooterMessageProps = OnboardingReportFooterMessageOnyxProps;
```

Also, let's add descriptions for each prop?